### PR TITLE
[14.0][mig][l10n_us_gaap, l10n_us_gaap_mis_report]

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Standard migration

@ForgeFlow

### CODE REVIEW

**Comment by ChrisOForgeFlow** [comment]
+1

**Comment by ChrisOForgeFlow** [comment]
+1

**Comment by JordiBForgeFlow** [comment]
@bodedra @dduarte-odoogap could you do a quick review?

**Review by ChrisOForgeFlow — APPROVED** [review]

**Comment by bodedra** [comment]
@JordiBForgeFlow 1 PR for 1 module?

**Comment by JordiBForgeFlow** [comment]
@bodedra one depends on the other. Since there are minimal changes I thought it would be an unnecessary overhead

File: l10n_us_gaap/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Provides a sample Chart of Accounts (CoA) for the United States based on the
+principles of GAAP.
+
+Additional remarks
+~~~~~~~~~~~~~~~~~~
+
+Companies should review and alter the sample CoA to fulfill the specific
+needs of their business.
+
+U.S. SEC not allow the classification of expenses by nature, thus the
```
Comment (bodedra) [inline l10n_us_gaap/readme/USAGE.rst:10]:
```suggestion
U.S. SEC does not allow the classification of expenses by nature, thus the
```

**Review by bodedra — APPROVED** [review]
Code review only. 👍

**Pull request merged**